Source/WebKit/UIProcess/Network/NetworkProcessProxy.h

```diff
@@ -31,6 +31,7 @@
 #include "DataReference.h"
 #include "DataTaskIdentifier.h"
 #include "IdentifierTypes.h"
+#include "LoadedWebArchive.h"
```

Comment:
We might be able to just forward declare LoadedWebArchive instead of including this here